ext/php_cassandra.c

```diff
@@ -173,12 +174,67 @@ ZEND_GET_MODULE(cassandra)
 #endif
 
 static void
-php_cassandra_log(const CassLogMessage* message, void *data)
+php_cassandra_log(const CassLogMessage* message, void* data)
 {
-  fprintf(stderr, "php-driver | [%s] %s (%s:%d)\n",
-    cass_log_level_string(message->severity), message->message,
-    message->file, message->line
-  );
+  char log[MAXPATHLEN + 1];
+  uint log_length = 0;
+
+  (void)data;
+
+  uv_rwlock_rdlock(&CASSANDRA_G(log_lock));
+  if (CASSANDRA_G(log)) {
+    log_length = MIN(CASSANDRA_G(log_length), MAXPATHLEN);
+    memcpy(log, CASSANDRA_G(log), log_length);
+  }
+  uv_rwlock_rdunlock(&CASSANDRA_G(log_lock));
+
+  log[log_length] = '\0';
+
+  if (log_length > 0) {
+    int fd = -1;
+
+    if (!strcmp(log, "syslog")) {
+      php_syslog(LOG_NOTICE, "php-driver | [%s] %s (%s:%d)",
```

Comment:
I think the driver's severity != syslog's severity because there are more critical events that could happen on a system and you might not want php-driver stuff showing up in your error/critical level system logs. I could either way here, but I'm in favor of leaving as notice.